python/seldon_core/utils.py

```diff
@@ -27,19 +35,14 @@
 logger = logging.getLogger(__name__)
 
 
-ENV_MODEL_NAME = "PREDICTIVE_UNIT_ID"
-ENV_MODEL_IMAGE = "PREDICTIVE_UNIT_IMAGE"
-NONIMPLEMENTED_MSG = "NOT_IMPLEMENTED"
-
-model_name = os.environ.get(ENV_MODEL_NAME, f"{NONIMPLEMENTED_MSG}")
-image_name = os.environ.get(
-    ENV_MODEL_IMAGE, f"{NONIMPLEMENTED_MSG}:{NONIMPLEMENTED_MSG}"
-)
-
-
 def get_request_path():
+    model_name = get_model_name()
     if model_name == NONIMPLEMENTED_MSG:
         return {}
+    image_name = get_image_name()
+    assert (
```

Comment:
Why the extra assertion? Just falling back to non-implemented seemed to work fine.